pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -135,6 +135,14 @@ const annDynamicallyProvisioned = "pv.kubernetes.io/provisioned-by"
 // a volume for this PVC.
 const annStorageProvisioner = "volume.beta.kubernetes.io/storage-provisioner"
 
+// This annotation is added to a PVC that has been triggered by scheduler to
+// be dynamically provisioned. Its value is the name of the selected node.
+const annSelectedNode = "volume.alpha.kubernetes.io/selected-node"
+
+// If the provisioner name in a storage class is set to "kubernetes.io/no-provisioner",
+// then dynamic provisioning is not supported by the storage.
+const NotSupportedProvisioner = "kubernetes.io/no-provisioner"
```

Comment:
Not really, since it's a general agreement, I make it public in case it is potentially used in other places.

However, if it's really used elsewhere in the future (say used by storage plugins), then the pv controller should no longer be the proper place to define it. So agree to make it internal for now.